roles/utilities/tasks/swap.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: check for swap
+  stat: path=/var/swap.1
+  register: swap
+
+- name: allocate swap
+  command: /bin/dd if=/dev/zero of=/var/swap.1 bs=1M count=1024
```

Comment:
The default VM size is like ~30GB; I personally don't think 1GB needs to be doc'd but if you think so, go ahead.